docs/core_docs/docs/integrations/chat/google_palm.mdx

```diff
@@ -28,3 +29,97 @@ the model.
 import GooglePaLMExample from "@examples/models/chat/integration_googlepalm.ts";
 
 <CodeBlock language="typescript">{GooglePaLMExample}</CodeBlock>
+
+# ChatGoogleVertexAI
+
+LangChain.js supports Google Vertex AI chat models as an integration.
+It supports two different methods of authentication based on whether you're running
+in a Node environment or a web environment.
+
+## Setup
+
+### Node
+
+To call Vertex AI models in Node, you'll need to install [Google's official auth client](https://www.npmjs.com/package/google-auth-library) as a peer dependency.
```

Comment:
Unless something changed, this is a dependency for @langchain/google-gauth so doesn't need to be a peer dependency.